Tasks/SonarQubePostTest/SonarQubePostTest.ps1

```diff
@@ -2,6 +2,10 @@ Write-Verbose "Starting SonarQube PostBuild Step"
 
 import-module "Microsoft.TeamFoundation.DistributedTask.Task.Common"
 
+Write-Warning "We are transferring ownership of the SonarQube related build tasks to Sonar Source. Please replace these build tasks with the ones provided by Sonar Source. 
```

Comment:
I'd prefer. "The ownership of the SonarQube related build tasks is transferred to SonarSource". Also SonarSource is in one word. We'd need to have the link on their extension, also. Maybe we should involve Freddy / Tamas in this PR?